static/src/js/components/TextWindowActions/TextWindowActions.scss

```diff
@@ -43,9 +43,18 @@
       padding-top: $spacer*1.5;
     }
   }
-
+  // TODO I think some of these can be cleared out theyre copies of Access
```

Comment:
Not sure what you mean by this

Reply:
Yea that was a debugging line. Its irrelevent now with the design for the EDD landing page anyway